examples/column_view_datagrid/grid_cell/imp.rs

```diff
@@ -0,0 +1,39 @@
+use gtk::glib;
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+
+use gtk::BinLayout;
+use gtk::CompositeTemplate;
+
+#[derive(Debug, Default, CompositeTemplate)]
+#[template(file = "grid_cell.ui")]
+pub struct GridCell {
+    #[template_child]
+    pub name: TemplateChild<gtk::Label>,
+}
+
+#[glib::object_subclass]
+impl ObjectSubclass for GridCell {
+    const NAME: &'static str = "GridCell";
+    type Type = super::GridCell;
+    type ParentType = gtk::Widget;
+
+    fn class_init(klass: &mut Self::Class) {
+        klass.set_layout_manager_type::<BinLayout>();
```

Comment:
Would add a quick note saying that we should either set a layout manager or override the `size_allocate` / `measure` functions of the `WidgetImpl` trait.

Reply:
added a quite note for this! let me know if that covers it